app/controllers/search_controller.rb

```diff
@@ -134,8 +133,9 @@ def feed_content
         )
       elsif class_name.Article?
         search_postgres_article
+      elsif class_name.Tag?
+        Search::Tag.search_documents(feed_params[:search_fields])
```

Comment:
I haven't implemented pagination because I assumed there shouldn't be too many `supported` tags. But now I think we should have it just in case :thinking: